File: 3408-count-the-number-of-special-characters-i/count-the-number-of-special-characters-i.py
class Solution:
    def numberOfSpecialChars(self, word: str) -> int:
        seen = [False] * 58
        for letter in word:
            seen[ord(letter) - ord('A')] = True
        specialChars = 0
        for letter in string.ascii_lowercase:
            if seen[ord(letter) - ord('A')] and seen[ord(letter.upper()) - ord('A')]:
                specialChars += 1
        return specialChars

File: 3408-count-the-number-of-special-characters-i/count-the-number-of-special-characters-i.go
func numberOfSpecialChars(word string) int {
    var seen [58]bool
    for _, c := range word {
        seen[c - 'A'] = true
    }
    specialChars := 0
    for i := 0; i < 26; i++ {
        if seen[i] && seen[i + 32] {
            specialChars += 1
        }
    }
    return specialChars
}